css/style.css

```diff
@@ -281,8 +390,8 @@ footer {
     gap: 15px;
 }
 
-.homepage-btn {         /* Do we want padding: 10px 20px; */
-    padding: 20px;
+.homepage-btn {
+    padding:10px 20px;
```

Comment:
This makes the buttons look a lot more uniform -- less square, slightly more rounded rectangle. This was a note left for the CSS overhaul so that we could have the same style/shape of buttons throughout the website. Thanks for adding it!